docs/cloud-workload-security/agent_expressions.md

```diff
@@ -16,14 +16,14 @@ Cloud Security Management Threats (CSM Threats) first evaluates activity within
 
 
 {{< code-block lang="javascript" >}}
-<event-type>.<event-attribute> <operator> <value> <event-attribute> ...
+<event-type>.<event-attribute> <operator> <value> <event-type>.<event-attribute> ...
```

Comment:
I would update it to something like `<event-type>.<event-attribute> <operator> <value> [<operator> <event-type>.<event-attribute>]...`